### DISCUSSION TITLE
Ability to copy/paste entire vector

### DISCUSSION THREAD

**Discussion by AndHalfAFish** [godotengine/godot-proposals#4292] [Editor]
As the editor is now, updating eg. the location of a node in 2 related scenes requires many mouse clicks. If it was possible to copy an entire vector, this would greatly reduce the amaount of work.

![copy_vector](https://user-images.githubusercontent.com/59080934/160289750-b9fd09fd-94d8-4c64-b541-ebbea823fe51.png)

**Comment by fire-forge** [comment]
In 4.0, properties have a context menu (right-click menu) that has a "Copy Property" button that does exactly that. I believe it hasn't been added to 3.x yet, but I could be wrong. It would just need to be backported by someone.

EDIT: It's in 3.5

**Reply by Mickeon** [reply]
It's not in 3.x yet, but it looks scheduled to be backported: https://github.com/godotengine/godot/pull/39404

**Reply by me2beats** [reply]
This is already in 3.5beta3

**Reply by lentsius-bark** [reply]
That is epic, had no idea! Thanks!
